src/rpc/RPCHelpers.cpp

```diff
@@ -720,8 +720,8 @@ traverseOwnedNodes(
                 return Status(
                     ripple::rpcINVALID_PARAMS, "Owner directory not found");
 
-            ripple::SerialIter it{ownerDir->data(), ownerDir->size()};
-            ripple::SLE sle{it, currentIndex.key};
+            ripple::SerialIter _it{ownerDir->data(), ownerDir->size()};
+            ripple::SLE _sle{it, currentIndex.key};
 
             for (auto const& key : sle.getFieldV256(ripple::sfIndexes))
```

Comment:
You need to be very very careful. This looks like it's now using the wrong `sle`... and you really gotta check all the code you changed in this PR so that it's not suddenly using the wrong variable. This is exactly one of the dangers of shadowing and why we try to fix this in the first place